### ISSUE TITLE
Code blocks in Markdown get actually rendered when wrapping them using ```html ... ```

### ISSUE THREAD

**Issue by sdellis** [viljamis/vue-design-system#135]
In the `/config/docs.config.js` file, when `exampleMode` and `usageMode` are set to "collapse" the "Vue|HTML" toggle should not display. Here is an example of a custom .md file that I've added.

```
...
    {
      name: "How to Install",
      content: "../docs/how-to-install.md",
      exampleMode: "collapse",
      usageMode: "collapse",
      sectionDepth: 1,
    },
...
```
<img width="701" alt="screen shot 2018-10-31 at 2 22 59 pm" src="https://user-images.githubusercontent.com/609368/47809981-9004fe00-dd18-11e8-811b-ae5f0b34578c.png">

**Comment by sdellis** [comment]
I should mention that in a previous version of VueDS one was able to add codeblocks that did not try to render by wrapping them in ````html`. That was helpful for providing installation instructions and other code samples in the docs (that have nothing to do with the components) without having it try to render the code.

**Comment by viljamis** [comment]
@sdellis html code blocks don’t seem to be be working currently. I think this might be a vue styleguidist issue, but not sure yet. This works:

```
    ```javascript
         something
    ```
```

but this seems to cause things to actually render on page and throw vue related errors:

```
    ```html
         <template>something</template>
    ```
```

**Comment by viljamis** [comment]
@rafaesc Would you have any clue on what’s going on?

**Comment by viljamis** [comment]
@sdellis Edited the title accordingly since I think that is the causing the tab issue as well.

**Comment by viljamis** [comment]
@sdellis I’ve created an issue on Styleguidist side about this: https://github.com/vue-styleguidist/vue-styleguidist/issues/234

**Comment by sdellis** [comment]
@viljamis Thank you! Should we leave this issue open until it gets resolved?

**Comment by viljamis** [comment]
@sdellis looks like there is a fix now in Styleguidist. If you want you can just update the version in your package.json, but please leave this open for now. I’ll make a new release soon with this and a few other fixes.

**Comment by viljamis** [comment]
Fixed in 3.5.7
